paths/uploads/create.yaml

```diff
@@ -85,6 +85,11 @@ requestBody:
             description: Indicates whether existing translations should be updated with the file content.
             type: boolean
             example:
+          update_translation_keys:
+            description: Indicates whether any missing translation keys should be created or existing ones updated during upload.
```

Comment:
I'd rather phrase it as "pass `false` here to prevent new keys from being created and existing keys updated" or similar.